module-integration.adoc

```diff
@@ -1,84 +1,79 @@
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+== Integrating The Raincatcher-GPS Module
+
 
 There are several changes required to integrate the raincatcher-gps
 module into the Demo Project:
 
-* Update the raincatcher-demo-mobile app to add the new raincatcher-gps
-module.
-* Update the raincatcher-user module to propagate user location updates
-to the Cloud.
-* Update the raincatcher-demo-auth app to handle location update topics.
-* Update the raincatcher-map module to handle displaying user locations
-on the map.
-* Update the raincatcher-demo-portal application to pass users to the
-raincatcher-map scope.
+. <<update-the-raincatcher-demo-mobile-app-to-add-the-new-raincatcher-gps-module,Update the raincatcher-demo-mobile app to add the new raincatcher-gps module>>.
+. <<update-the-raincatcher-user-module-to-propagate-user-location-updates-to-the-cloud,Update the raincatcher-user module to propagate user location updates to the Cloud>>.
+. <<update-the-raincatcher-demo-auth-app-to-handle-location-update-topics,Update the raincatcher-demo-auth app to handle location update topics>>.
+. <<update-the-raincatcher-map-module-to-handle-displaying-user-locations-on-the-map,Update the raincatcher-map module to handle displaying user locations on the map>>.
+. <<update-the-raincatcher-demo-mobile-app-to-add-the-new-raincatcher-gps-module,Update the raincatcher-demo-portal application to pass users to the raincatcher-map scope>>.
 
 [[update-the-raincatcher-demo-mobile-app-to-add-the-new-raincatcher-gps-module]]
-1. Update the raincatcher-demo-mobile app to add the new raincatcher-gps
-module
-+
+
+===  1. Update the raincatcher-demo-mobile app to add the new raincatcher-gps module
+
 The goal of this change is to capture the location of mobile workers in
 the field as they complete work orders. To do this, we can add the
 raincatcher-gps module to the raincatcher-gps module by:
 
-[[a.-include-the-raincatcher-gps-module-in-the-main-app-dependencies]]
-  .. Include the raincatcher-gps module in the main app dependencies
+[[include-the-raincatcher-gps-module-in-the-main-app-dependencies]]
+a. Include the raincatcher-gps module in the main app dependencies
 +
 Run `npm install fh-wfm-gps --save` to install the raincatcher-gps
 module and automatically add it to the dependencies in the
 `package.json` file.
 +
 In the `src/app.js` file, the `fh-wfm-gps` module is included in the
 main angular dependencies.
-
-[[b.-create-a-new-service-to-publish-and-subscribe-to-location-events]]
-  .. Create a new service to publish and subscribe to location events
++
+[[create-a-service-to-publish-and-subscribe-to-location-events]]
+a. Create a service to publish and subscribe to location events
 +
 The raincatcher-mobile app is responsible for managing the location
 based messages passing between the raincatcher-gps module and
 raincatcher-user module.
 +
+//TODO: very passive below, what is happening?
```

Comment:
removed from this file, now all I need is a todo to remove them from the other files ;)